AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/Annotation.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import CoreGraphics
+
+public final class Annotation: Codable {
+    public let id: UUID?
+    public private(set) var origin: CGPoint
+    public private(set) var width: Double
+    public let ownerId: UUID
```

Comment:
```suggestion
    public let ownerId: String
```

Should this be String instead? Same reason as for documents